modules/commons/src/main/java/org/apache/synapse/commons/json/JsonUtil.java

```diff
@@ -454,12 +500,32 @@ public static void transformElement(OMElement element, boolean processAttrbs) {
         }
     }
 
+    /**
+     * Removes XML namespace declarations, and namespace prefixes from an XML element.
+     *
+     * @param requiredProperties Message context properties
+     * @param element            Source XML element
+     * @param processAttrbs      Whether to remove the namespaces from attributes as well
+     */
+    public static void transformElement(Map requiredProperties, OMElement element, boolean processAttrbs) {
```

Comment:
let's reorder function signature. Give priority to important arguments.

Reply:
Done